realm/realm-library/src/main/cpp/io_realm_internal_TableQuery.cpp

```diff
@@ -61,6 +62,31 @@ JNIEXPORT jstring JNICALL Java_io_realm_internal_TableQuery_nativeValidateQuery(
 
 // helper functions
 
+/*static TableRef resolve_link(jlong nativeQueryPtr, JniLongArray& fields) {
```

Comment:
ewwwwww!  If we don't need this, let's delete it.  Remove the evidence of my unclear thinking, please.  No commented out code.